cmd/ocm-backplane/login/login.go

```diff
@@ -324,7 +333,7 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 
 	targetContext.AuthInfo = targetUserNickName
 	targetContext.Cluster = clusterName
-	targetContext.Namespace = "default"
+	targetContext.Namespace = args.defaultNamespace
```

Comment:
Small suggestion,  `args.defaultNamespace` may need to validate for empty NS

Reply:
Thank you.  I can make validation more robust  by ensuring the following [statement](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns) is satisfied.
_As a result, all namespace names must be valid [RFC 1123 DNS labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)._

Reply:
https://github.com/openshift/backplane-cli/pull/374/commits/0560e9731307617ad6feca43106fe01d2d55b398 accomdates your comment.